homeassistant/components/hue/sensor_base.py

```diff
@@ -47,7 +47,7 @@ class SensorManager:
     Intended to be a singleton.
     """
 
-    SCAN_INTERVAL = timedelta(seconds=5)
+    SCAN_INTERVAL = timedelta(seconds=0.1)
```

Comment:
This short scan interval will never be approved.

Reply:
I simply reported the value of robmarkcole
Plugin widely used in the community

https://raw.githubusercontent.com/robmarkcole/Hue-sensors-HASS/master/custom_components/huesensor/sensor.py

line 23
SCAN_INTERVAL = timedelta (seconds = 0.1)

the 5-second interval does not detect long presses on the switch
I can better set the interval to 500ms

Is a value less than one second acceptable ?

On developers site
https://developers.meethue.com/develop/application-design-guidance/hue-system-performance/

#ZigBee messages per hue API command | Parameters used in body of hue API command (example) | System throughput | Time between hue API messages
-- | -- | -- | --
1 | setting brightness | brightness | 40 ms
1 | setting brightness+color | 25 hue API commands/s | 80 ms
3 | setting brightness + color+on | 2 hue API commands/s | 120 ms

Reply:
Martin is correct. We will never go lower than 5.